Change installation script according to Debian standards.
The -qt4 ending was originally used to make qt3 and qt4
version coinstallable.
--- a/designer/designer.pro
+++ b/designer/designer.pro
@@ -61,7 +61,7 @@
         DEFINES += NO_QWT_WIDGETS
     }
 
-    unix:LIBS      += -L$${QWT_ROOT}/lib -l$${LIBNAME}
+    unix:LIBS      += -L$${QWT_ROOT}/lib -l$${LIBNAME}-qt4
     win32-msvc:LIBS  += $${QWT_ROOT}/lib/$${LIBNAME}.lib
     win32-msvc.net:LIBS  += $${QWT_ROOT}/lib/$${LIBNAME}.lib
     win32-msvc2002:LIBS += $${QWT_ROOT}/lib/$${LIBNAME}.lib
@@ -126,7 +126,7 @@
         RESOURCES += \
             qwt_designer_plugin.qrc
 
-        target.path = $$[QT_INSTALL_PLUGINS]/designer
+        target.path = /usr/lib/qt4/plugins/designer
         INSTALLS += target
     }
 }
--- a/examples/examples.pri
+++ b/examples/examples.pri
@@ -35,8 +35,8 @@
 TEMPLATE     = app
 
 MOC_DIR      = moc
-INCLUDEPATH += $${QWT_ROOT}/src
-DEPENDPATH  += $${QWT_ROOT}/src
+INCLUDEPATH += /usr/include/qwt-qt4
+DEPENDPATH  += /usr/include/qwt-qt4
 OBJECTS_DIR  = obj$${SUFFIX_STR}
 DESTDIR      = $${QWT_ROOT}/examples/bin$${SUFFIX_STR}
 
@@ -57,5 +57,5 @@
     win32-g++:LIBS   += -L$${QWT_ROOT}/lib -l$${QWTLIB}
 }
 else {
-    LIBS        += -L$${QWT_ROOT}/lib -l$${QWTLIB}
+    LIBS        += -L$${QWT_ROOT}/lib -l$${QWTLIB}-qt4
 }
--- a/examples/examples.pro
+++ b/examples/examples.pro
@@ -7,7 +7,8 @@
 # modify it under the terms of the Qwt License, Version 1.0
 ###################################################################
 
-include( ../qwtconfig.pri )
+include( examples.pri )
+include( qwtconfig.pri )
 
 TEMPLATE = subdirs
 
--- a/qwtconfig.pri
+++ b/qwtconfig.pri
@@ -8,7 +8,7 @@
 VERSION      = $${VER_MAJ}.$${VER_MIN}.$${VER_PAT}
 
 unix {
-    INSTALLBASE    = /usr/local/qwt-$$VERSION
+    INSTALLBASE    = /usr
 }
 
 win32 {
@@ -16,8 +16,8 @@
 }
 
 target.path    = $$INSTALLBASE/lib
-headers.path   = $$INSTALLBASE/include
-doc.path       = $$INSTALLBASE/doc
+headers.path   = $$INSTALLBASE/include/qwt-qt4
+doc.path       = $$INSTALLBASE/share/doc/libqwt5-doc
 
 ######################################################################
 # qmake internal options
@@ -98,7 +98,7 @@
 # QwtSVGItem.
 ######################################################################
 
-#CONFIG     += QwtSVGItem
+CONFIG     += QwtSVGItem
 
 ######################################################################
 # You can use the MathML renderer of the Qt solutions package to 
--- a/src/src.pro
+++ b/src/src.pro
@@ -34,7 +34,7 @@
     }
 }
 
-TARGET            = qwt$${SUFFIX_STR}
+TARGET            = qwt$${SUFFIX_STR}-qt4
 TEMPLATE          = lib
 
 MOC_DIR           = moc
